hello-jni/build.gradle

```diff
@@ -15,6 +16,7 @@ buildscript {
 allprojects {
     repositories {
         jcenter()
```

Comment:
Put google() first, before jcenter().

Believe it or not, certain things features, such as like running Lint checks on Kotlin code from the command line, doesn't work unless you include google() first (don't ask me why).

Reply:
done, will change other samples too